src/Language/PureScript/CodeGen/Externs.hs

```diff
@@ -38,11 +38,24 @@ import Language.PureScript.Types
 moduleToPs :: Module -> Environment -> String
 moduleToPs (Module _ _ _ Nothing) _ = error "Module exports were not elaborated in moduleToPs"
 moduleToPs (Module _ moduleName ds (Just exts)) env = intercalate "\n" . execWriter $ do
-  tell [ "module " ++ runModuleName moduleName ++ " where"]
+  let exps = listExports exts
+  tell ["module " ++ runModuleName moduleName ++ (if null exps then "" else " (" ++ exps ++ ")") ++ " where"]
```

Comment:
I think this is my fault actually, but can we change this to `if null exps then " ()" ...`?